package.json

```diff
@@ -20,6 +20,7 @@
     "node": ">=14.x"
   },
   "dependencies": {
+    "@chec/commerce.js": "^2.8.0",
```

Comment:
I realized that this sdk uses axios internally which is a pretty heavy lib, but its not being used a lot, probably using other fetch library might benefit the sdk also. This is not a hard requirement but its highly advised as currently all dependencies are installed for all providers (but just the ones needed are used)

Reply:
The SDK is actually doing all the data fetching on both client and server-side, in the two "fetcher" functions which call SDK methods:

https://github.com/cpv123/commerce/blob/commercejs-provider/framework/commercejs/api/utils/sdk-fetch.ts
https://github.com/cpv123/commerce/blob/commercejs-provider/framework/commercejs/fetcher.ts

So I wouldn't want to remove the SDK dependency otherwise these fetcher functions would have to be re-written to make HTTP requests directly, and the benefits of using the SDK would be lost (such as standardised responses, errors, typings, etc.) Hopefully, this is OK for now.

There is a new version of the Commerce.js SDK being planned, and I'll make sure to leave some feedback about axios and the possibility of using a different (lighter) fetch library.

Reply:
Commerce.js dev here! Yep the axios inclusion was a pretty old decision. We're planning to switch to fetch with the new version of Commerce.js.